### ISSUE TITLE
Differentiate ClinVar variants with conflicting interpretations of pathogenicity

### ISSUE THREAD

**Issue by nawatts** [broadinstitute/gnomad-browser#751]
Currently, ClinVar variants with multiple interpretations are all labeled as "Conflicting interpretations of pathogenicity". On the variant page, details about each submission can be seen. On the gene/region/transcript pages, however, there's no way to get more detail about the conflict.

The ClinVar track on these pages should include whether the variant is "Conflicting (P/LP vs. VUS/LB/B)” or “Conflicting (VUS vs. LB/B)".

**Comment by nawatts** [comment]
https://atgu.slack.com/archives/CNLMF0JLV/p1617201073047100

**Comment by Shicheng-Guo** [comment]
Hi Nick, 

I download VCF files and try to find "likely pathogenic, Conflicting interpretations of pathogenicity" information, however, it looks ClinVar annotation is not existed in download VCF, right? 

Thanks. 

Shicheng

**Comment by nawatts** [comment]
> ClinVar annotation is not existed in download VCF, right?

Correct. The ClinVar data displayed in the browser is downloaded from ClinVar about once a month. It is not included in the gnomAD release files.

https://www.ncbi.nlm.nih.gov/clinvar/docs/maintenance_use/#download
https://ftp.ncbi.nlm.nih.gov/pub/clinvar/xml/clinvar_variation/ClinVarVariationRelease_00-latest.xml.gz
